### ISSUE TITLE
Error on IE 8 "This comment is not supported"

### ISSUE THREAD

**Issue by fedegiust** [t4t5/sweetalert#329]
Hi @t4t5 ,

I'm testing your plugin which is quite cool, so far I love it.

But I'm facing an issue on IE8 (I know, some of our client still use it...), but anyways the error is

Webpage error details

User Agent: Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 6.1; WOW64; Trident/4.0; SLCC2; .NET CLR 2.0.50727; .NET CLR 3.5.30729; .NET CLR 3.0.30729; .NET4.0C; .NET4.0E)
Timestamp: Fri, 10 Apr 2015 16:07:24 UTC

Message: This command is not supported.
Line: 861
Char: 5
Code: 0
URI: http://localhost/content/vendor/sweetalert/lib/sweet-alert.js

This is the function in question

```
  function resetInput() {
      var $modal = getModal();
     var $input = getInput();

removeClass($modal, 'show-input');
$input.value = "";
**$input.setAttribute('type', defaultParams.inputType);** //this is the line

swal.resetInputError();
  }
```

This is the code I'm using to show up the message

```
            ``  <script type="text/javascript">
                swal({
                                title: 'Invalid Access!',
                                text: 'Error: User Name or Password not found or invalid.',
                               type: 'error'
                });
                </script>``
```

This works fine on all other browsers I tested (Chrome, Safari, Firefox)

Thanks in advance

**Comment by DeanPoulin** [comment]
:+1:

**Comment by bryce-gibson** [comment]
IE8 doesn't let you set the `type` of an input using `setAttribute`.

To fix it, it's possible to use `$input.type = defaultParams.inputType;` instead.

There's a 0.5.x branch here https://github.com/bryce-gibson/sweetalert/tree/0.5.x that has this fix, if helpful.

**Comment by GreatWizard** [comment]
Thank you @bryce-gibson, it works perfectly!

Do you think you can bring your fixed to @t4t5 to merge in its branch 0.5 to have an "official" 0.5.1 available on npm ?

**Comment by bryce-gibson** [comment]
More than happy for it to be merged into a 0.5.x type branch, but I'm not sure how to make a PR to a new branch in Github? :-s

(If there is a 0.5.x type branch, I'd be more than happy to help - as we're using Sweetalert with IE8 here, so we're fixing any issues we're encountering...)

**Comment by t4t5** [comment]
I think it's time to say goodbye to IE8, so I'm closing this issue. :)
